stubs/django/test/client.pyi

```diff
@@ -1 +1,84 @@
+from typing import Any, Dict, Optional
+
+from django.http.response import HttResponse
```

Comment:
The `Annotation 'HttpResponse' is not defined as a type.` are because of the typo in this import